scss/individual/base.scss

```diff
@@ -0,0 +1,2 @@
+@import '../base';
```

Comment:
We are not using this file in any example directly, but we can keep it to make sure base styles can be built on their own.